lib/pages/chat_list/chat_list_item_title.dart

```diff
@@ -40,26 +40,28 @@ class ChatListItemTitle extends StatelessWidget with ChatListItemMixin {
             children: [
               Row(
                 children: [
+                  Flexible(
+                    child: Padding(
+                      padding: ChatListItemTitleStyle.paddingRightTitle,
+                      child: Text(
+                        displayName,
+                        overflow: TextOverflow.ellipsis,
+                        maxLines: 1,
+                        softWrap: false,
+                        style: textStyle ??
+                            ChatLitTitleTextStyleView.textStyle.textStyle(room),
+                      ),
+                    ),
+                  ),
                   if (room.encrypted)
                     Padding(
-                      padding:
-                          const EdgeInsets.only(right: 4, top: 2, bottom: 2),
+                      padding: ChatListItemTitleStyle.paddingLeftIcon,
```

Comment:
Padding left is 7px, pls update

<img width="969" alt="Screenshot 2024-09-16 at 09 26 36" src="https://github.com/user-attachments/assets/bebe57b4-1be5-43eb-9150-4a2a0b830167">

Reply:
The icon has 4 padding  to the left, and the title has 3 to the right, making it a total of 7, so it's implemented correctly